data/categories.txt

```diff
@@ -2817,7 +2817,7 @@ sk:3Vyhliadka
 sw:Genge|utalii
 fa:مناظرگردشگری
 
-tourism-information|@tourism
+tourism-information
```

Comment:
Does it include only information offices or also information boards? Offices do not pollute results much and are valuable for tourists. Are there many boards in touristic cities? Do they pollute search results? Can we distinguish them with offices?